crowbar_framework/app/models/ceilometer_service.rb

```diff
@@ -32,21 +32,33 @@ def role_constraints
       {
         "ceilometer-agent" => {
           "unique" => false,
-          "count" => -1
+          "count" => -1,
+          "exclude_platform" => {
+            "suse" => "12.0"
```

Comment:
I will fix it. In the task description this was not noted, but can be deduced becasue Nova can go there.